web/profiles/custom/yalesites_profile/config/sync/core.entity_form_display.paragraph.custom_cards.default.yml

```diff
@@ -0,0 +1,58 @@
+uuid: 69be657d-9967-4b62-a4d9-4f37a426a2a2
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.custom_cards.field_cards
+    - field.field.paragraph.custom_cards.field_heading
+    - paragraphs.paragraphs_type.custom_cards
+  module:
+    - paragraphs
+    - paragraphs_ee
+    - paragraphs_features
+    - text
+id: paragraph.custom_cards.default
+targetEntityType: paragraph
+bundle: custom_cards
+mode: default
+content:
+  field_cards:
+    type: paragraphs
```

Comment:
I'm not sure if it's this value or not, but something in this file should change. You are using "Paragraphs (stable)" but other components are using "Paragraphs Legacy" (see accordion and callout which both have child items.) Please change this to use the legacy version for consistency. I know, it's a nit picky thing that may change in the future.